quiche/src/lib.rs

```diff
@@ -19100,45 +19096,23 @@ mod tests {
         )
         .unwrap();
 
-        let pmtud_enabled = pipe
-            .server
-            .paths
-            .get_active_mut()
-            .unwrap()
-            .pmtud
-            .is_enabled();
-        assert_eq!(pmtud_enabled, false);
+        {
+            let active_path = pipe.server.paths.get_active_mut().unwrap();
+            assert!(!active_path.pmtud.is_enabled());
+        }
 
         assert_eq!(pipe.handshake(), Ok(()));
 
-        let pmtud_enabled = pipe
-            .server
-            .paths
-            .get_active_mut()
-            .unwrap()
-            .pmtud
-            .is_enabled();
-        assert_eq!(pmtud_enabled, true);
-
-        let current_path_size = pipe
-            .server
-            .paths
-            .get_active_mut()
-            .unwrap()
-            .pmtud
-            .get_current();
-        assert_eq!(current_path_size, 1200);
+        {
+            let active_path = pipe.server.paths.get_active_mut().unwrap();
+            assert!(active_path.pmtud.is_enabled());
+            assert_eq!(active_path.pmtud.get_current_mtu(), 1200);
+        }
 
         assert_eq!(pipe.advance(), Ok(()));
 
-        let new_path_size = pipe
-            .server
-            .paths
-            .get_active_mut()
-            .unwrap()
-            .pmtud
-            .get_current();
-        assert_eq!(new_path_size, 1350);
+        let active_path = pipe.server.paths.get_active_mut().unwrap();
+        assert_eq!(active_path.pmtud.get_current_mtu(), 1350);
```

Comment:
Consider adding {}'s around this block for consistency.

Reply:
Done